zebra-network/src/peer/handshake.rs

```diff
@@ -380,7 +380,7 @@ where
                         if msg.is_ok() {
                             // XXX add a dimension tagging message metrics by type
                             metrics::counter!(
-                                "inbound_messages",
+                                "zcash.net.in.messages",
```

Comment:
I was confused when I came to make this change, because I was sure this was previously `peer.inbound_messages` (matching outbound).

Reply:
Hmm yeah that is odd, but it looks like the commit that added it was inconsistent:
https://github.com/ZcashFoundation/zebra/commit/80e7ee6daeec3dd64d35909426781fa678e5e914